utils/misc/misc.go

```diff
@@ -45,6 +45,10 @@ import (
 
 var AppStartTime int64
 var errorStorePath string
+var jobQueryBatchSize int
+var RouterInMemoryJobCounts map[string]map[string]map[string]int
+var routerJobCountMutex sync.RWMutex
+var ProcessorJobsMovingAverages map[string]map[string]map[string]MovingAverage
```

Comment:
I would suggest avoid introducing more global variables and instead create a struct will contain the variables and logic necessary for stats to work.

Also, I don't think `utils/misc` is the correct package to host such a rich functionality, instead a new `service/tenantstats` could be used.